x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -341,6 +350,40 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
           />
         </EuiFlexItem>
       </ProgressControls>
+      {showLogRateAnalysisResultsTable && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiCallOut
+            title={
+              <span data-test-subj="aiopsAnalysisTypeCalloutTitle">
+                {currentAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+                  ? i18n.translate('xpack.aiops.analysis.analysisTypeSpikeCallOutTitle', {
+                      defaultMessage: 'Analysis type: Log rate spike',
+                    })
+                  : i18n.translate('xpack.aiops.analysis.analysisTypeDipCallOutTitle', {
+                      defaultMessage: 'Analysis type: Log rate dip',
+                    })}
+              </span>
+            }
+            color="primary"
+            iconType="pin"
+            size="s"
+          >
+            <EuiText size="s">
+              {currentAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+                ? i18n.translate('xpack.aiops.analysis.analysisTypeSpikeCallOutContent', {
+                    defaultMessage:
+                      'The median log rate in the selected deviation time range is higher than the baseline. Therefore, the results shows statistically significant items within the deviation time range that are contributors to the spike. The "doc count" column refers to the amount of documents in the deviation time range.',
+                  })
+                : i18n.translate('xpack.aiops.analysis.analysisTypeDipCallOutContent', {
+                    defaultMessage:
+                      'The median log rate in the selected deviation time range is lower than the baseline. Therefore, the analysis results table shows statistically significant items within the baseline time range that are less present or missing within the deviation time range. The "doc count" column refers to the amount of documents in the baseline time range.',
```

Comment:
`less in number` better than `less present`? Any thoughts @szabosteve ?

Reply:
Updated in 4f8c71ac387.

Reply:
Sorry, I'm late to the party! Yes, `less in number` is a clearer solution. Thanks for updating!